<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Input1</class>
 <widget class="QDialog" name="Input1">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计算器</string>
  </property>
  <widget class="QLabel" name="CalculateName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>傅里叶空间像素大小</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="mLable">
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>    m:    </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mInput">
       <property name="maximumSize">
        <size>
         <width>265</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mUnit">
       <property name="maximumSize">
        <size>
         <width>37</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>  pcs  </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="nLabel">
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>     n:    </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nInput">
       <property name="maximumSize">
        <size>
         <width>265</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nUnit">
       <property name="text">
        <string>  pcs  </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="pnLabel">
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>   pn:    </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pnInput">
       <property name="maximumSize">
        <size>
         <width>265</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pnUnit">
       <property name="text">
        <string>   Å   </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="pmLabel">
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>   pm:   </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pmInput">
       <property name="maximumSize">
        <size>
         <width>265</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pmUnit">
       <property name="text">
        <string>   Å   </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="CalculateBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
